lib/active_shipping/package_item.rb

```diff
@@ -61,7 +60,7 @@ def kilograms(options = {})
 
     def attribute_from_metric_or_imperial(obj, klass, metric_unit, imperial_unit)
       if obj.is_a?(klass)
-        return value
+        return obj
```

Comment:
Oh great. Yeah I have some tests that I added after to specify behaviour that isn't used.

To be real, for 2.0 we should kill most of those code paths anyway.